### PR TITLE
docs: update security information and readme links

### PR DESCRIPTION
**Pull request by kpanot**
## Related issues

- :rocket: Feature #1502

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,3 +222,11 @@ Example:
   }
 }
 ```
+
+#### SSL Certificate issue (behind proxy)
+
+Due to proxy redirection you may face SSL certificate issue on Yarn when installing the Otter project locally.
+To solve it you can provide your own certificate via 2 different ways:
+
+* By providing you certificate path in the Environment Variable [NODE_EXTRA_CA_CERTS](https://nodejs.org/docs/latest-v4.x/api/cli.html#cli_node_extra_ca_certs_file).
```
Comment (fpaul-1A) [inline README.md:231]:
```suggestion
* By providing your certificate path in the Environment Variable [NODE_EXTRA_CA_CERTS](https://nodejs.org/docs/latest-v4.x/api/cli.html#cli_node_extra_ca_certs_file).
```

File: README.md [hunk 2]
```diff
@@ -220,3 +222,11 @@ Example:
   }
 }
 ```
+
+#### SSL Certificate issue (behind proxy)
+
+Due to proxy redirection you may face SSL certificate issue on Yarn when installing the Otter project locally.
```
Comment (fpaul-1A) [inline README.md:228]: you may add a quote of the type of error you can get to help people find this with search tools
> Request Error: self-signed certificate in certificate chain

**Review by fpaul-1A — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -13,9 +13,10 @@ It is split into several units to cover different aspects of these applications
 Also, to customize an application, metadata can be extracted from the application source code and injected into a CMS to manage dynamic configuration.
 
 > [!TIP]
-> The full documentation is available [here](./docs/README.md) and the Architecture overview schema is available [here](./docs/core/ARCHITECTURE.md).
+> The full list of modules and their documentations is available in [/docs folder](./docs/README.md).
```
Comment (sdo-1A) [inline README.md:16]:
```suggestion
> The full list of modules and their documentation is available in the [/docs folder](./docs/README.md).
```

File: README.md [hunk 4]
```diff
@@ -13,9 +13,10 @@ It is split into several units to cover different aspects of these applications
...
+> A big picture of the technical module interactions and dependencies architecture is available in the [architecture section](./docs/core/ARCHITECTURE.md).
```
Comment (sdo-1A) [inline README.md:17]:
```suggestion
> An overview of the technical module interactions and the architecture of the dependencies is available in the [architecture section](./docs/core/ARCHITECTURE.md).
```

File: README.md [hunk 5]
```diff
@@ -220,3 +222,16 @@ Example:
   }
 }
 ```
+
+#### SSL Certificate issue (behind proxy)
+
+Due to proxy redirection you may face SSL certificate issue on Yarn when installing the Otter project locally:
```
Comment (sdo-1A) [inline README.md:228]:
```suggestion
Due to proxy redirection you may face an SSL certificate issue on Yarn when installing the Otter project locally:
```

File: README.md [hunk 6]
```diff
@@ -220,3 +222,16 @@ Example:
...
+
+```
+Request Error: self-signed certificate in certificate chain
+```
+
+To solve it you can provide your own certificate via 2 different ways:
```
Comment (sdo-1A) [inline README.md:234]:
```suggestion
To solve this, you can provide your own certificate in two different ways:
```

File: README.md [hunk 7]
```diff
@@ -220,3 +222,16 @@ Example:
...
+
+* By providing your certificate path in the Environment Variable [NODE_EXTRA_CA_CERTS](https://nodejs.org/docs/latest-v4.x/api/cli.html#cli_node_extra_ca_certs_file).
```
Comment (sdo-1A) [inline README.md:236]:
```suggestion
* Specify your certificate path in the Environment Variable [NODE_EXTRA_CA_CERTS](https://nodejs.org/docs/latest-v4.x/api/cli.html#cli_node_extra_ca_certs_file).
```

File: README.md [hunk 8]
```diff
@@ -220,3 +222,16 @@ Example:
...
+* by providing your certificate path to the Yarn configuration [httpsCertFilePath](https://yarnpkg.com/configuration/yarnrc#httpsCertFilePath).
```
Comment (sdo-1A) [inline README.md:237]:
```suggestion
* Specify your certificate path in the Yarn configuration [httpsCertFilePath](https://yarnpkg.com/configuration/yarnrc#httpsCertFilePath).
```

File: SECURITY.md [hunk 9]
```diff
@@ -25,11 +25,12 @@ A major version will be supported as long as it relies on a [non deprecated vers
...
-|:---------------:|:-----------------:|:--------------:|
-|        9        |        16         |   2024-11-08   |
-|        8        |        15         |   2024-05-18   |
-|        7        |        14         |   2023-11-18   | 
+| Otter version | Angular version | Support ends |
+| :-----------: | :-------------: | :----------: |
+|      10       |       17        |      -       |
+|       9       |       16        |  2024-11-08  |
+|       8       |       15        |  2024-05-18  |
+|       7       |       14        |  2023-11-18  |
```
Comment (sdo-1A) [inline SECURITY.md:33]: do we keep this version since it is no longer supported since 2023 ?

Comment (kpanot) [reply SECURITY.md:33]: I would say yes, removing them does not bring any value.
(I asked myself the same question)

**Review by cpaulve-1A — APPROVED** [review]

**Review by sdo-1A — APPROVED** [review]

**Pull request merged**